frontend/src/app/offers/create-offer/create-offer.component.ts

```diff
@@ -0,0 +1,80 @@
+import { Component, OnInit } from '@angular/core';
+import { Offer } from 'app/homepage-offer/offers.model';
+import { AuthService } from 'app/auth.service';
+import { User } from 'app/user';
+import { Observable } from 'rxjs/Observable';
+import 'rxjs/add/operator/map';
+import { OffersService } from 'app/homepage-offer/offers.service';
+import { ActivatedRoute, Params } from '@angular/router';
+import { environment } from 'environments/environment';
+
+@Component({
+  selector: 'volontulo-create-offer',
+  templateUrl: './create-offer.component.html',
+  styleUrls: ['./create-offer.component.scss']
+})
+export class CreateOfferComponent implements OnInit {
+  djangoRoot: string;
+  public inEditMode: boolean = false;
+  offer: Offer = new Offer;
+  user: User;
+  isAdmin: boolean = false;
+  hasOrganization = false;
+
+  constructor(
+    private authService: AuthService,
+    private offersService: OffersService,
+    private route: ActivatedRoute,
+  ) {
+      this.djangoRoot = environment.djangoRoot;
```

Comment:
environments are not subject to change, so this variable can be defined once per class, not per every object separately.